client/app/components/application.coffee

```diff
@@ -40,153 +30,58 @@ MessageActionCreator = require '../actions/message_action_creator'
     (based on:
         https://medium.com/react-tutorials/react-backbone-router-c00be0cf1592)
 ###
-module.exports = Application = React.createClass
+
+Application = React.createClass
     displayName: 'Application'
 
     mixins: [
-        StoreWatchMixin Stores
-        RouterMixin
         TooltipRefesherMixin
     ]
 
-    render: ->
-        # Shortcut
-        # TODO: Improve the way we display a loader when app isn't ready
-        layout = @props.router.current
-        return div null, t "app loading" unless layout?
-
-        disposition = LayoutStore.getDisposition()
-        isCompact   = LayoutStore.getListModeCompact()
-        fullscreen  = LayoutStore.isPreviewFullscreen()
-        previewSize = LayoutStore.getPreviewSize()
+    getDefaultProps: ->
+        ApplicationGetters.getProps 'application'
```

Comment:
We need to decide a guideline for getters : 
In my humble opinion : 
`getDefaultProps` and `PropTypes` belongs to the view
`getStateFromStore` should be explicit :
 ```coffee
getStateFromStore: ->
     stuff1 : SomeGetter.getStuff1()
     stuff2 : SomeGetter.getStuff2()
```
`ApplicationGetters.getMenuProps() ` rather than `ApplicationGetters.getProps('menu')` (which by the way should go away as Menu is a Controller)